docs/licensing.md

```diff
@@ -17,7 +20,22 @@ The current limitations are applied to the community version:
     You need to pass in a setting to Qlik Associative Engine to acknowledge that
     you accept the EULA. Example: `docker run qlikcore/engine:<version> -S AcceptEULA=yes`
 
-## Licensed
+## 2. Evaluation Trial License
+
+To better evaluate Qlik Core without the barriers in the developer edition, register to obtain a
+free 90-day license key.
+
+With this evaluation trial license, you can:
+
+* Avoid having the Qlik Associative Engine expire 30 days after the version has been released
```

Comment:
Can we refer to the developer edition limitation section instead? Easier to keep a consistent message if there's only one source :) Same under the "Licensed" notice referring to "community" edition.

Reply:
Removed this and focused on what you can do with the trial license.